src/libraries/Common/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -19,10 +21,10 @@ internal static partial class ADP
     {
         // NOTE: Initializing a Task in SQL CLR requires the "UNSAFE" permission set (https://docs.microsoft.com/en-us/dotnet/framework/performance/sql-server-programming-and-host-protection-attributes)
         // Therefore we are lazily initializing these Tasks to avoid forcing customers to use the "UNSAFE" set when they are actually using no Async features
-        private static Task<bool> _trueTask;
+        private static Task<bool>? _trueTask;
         internal static Task<bool> TrueTask => _trueTask ?? (_trueTask = Task.FromResult(true));
```

Comment:
Nit: if you wanted to clean this up further, this line could be:
```suggestion
        internal static Task<bool> TrueTask => _trueTask ??= Task.FromResult(true);
```
Same for the one below.

Reply:
There's tons of potential for cleanup (wait till you see the next PR here...), with the already-massive nature of these PRs I opted for changing the least possible and leave that for later... Will do your suggestion though.